malpolon/models/standard_prediction_systems.py

```diff
@@ -348,6 +355,7 @@ def predict_point(
         """
         device = 'cuda' if torch.cuda.is_available() else 'cpu'
         self.model.to(device)
+        data = data.to(device)
```

Comment:
`data` isn't guaranteed to be a Tensor since this method doesn't require it to be (intentional behavior since the method should be callable by hand on any data type).
If `data` isn't a Tensor, the calling of `.to()` will fail and crash.
The data (or its elements) are casted to `device` anyway in the following lines.

=> Recommendation: delete this line